.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
   - 2.2.3
+before_script:
+  - sed -f _scripts/baseurl.sed _config.yml > _scripts/_config.yml
+  - mv _scripts/_config.yml _config.yml 
```

Comment:
I actually liked the name `_config_deploy.yml` so it's clear to the reader why we're doing this. I also think it's useful for the reader to see in the `jekyll build` call that it's using `config_deploy.yml` explicitly.